gulpfile.js

```diff
@@ -57,27 +61,27 @@ gulp.task('coveralls', ['test'], function() {
 });
 
 gulp.task('compile', ['clean'], function() {
-  return browserify('./lib/bespoke-keys.js')
-    .bundle({ standalone: 'bespoke.plugins.keys' })
+  return browserify({debug: true, standalone: 'bespoke.plugins.keys'})
```

Comment:
Did you mean to leave debug enabled on browserify? I don't think we should have debug symbols in the distributed files.